docs/ruleset_functions/does_each_zone_have_only_one_terminal(hvac.id).md

```diff
@@ -0,0 +1,24 @@
+# does_each_zone_have_only_one_terminal   
+
+**Description:** Returns TRUE if each zone input to this function only has one terminal unit. It returns FALSE if any zone has more than one terminal unit.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if each zone input to this function only has one terminal unit.   
+- **zone_id_list**: List of zones to evaluate.  
+
+**Returns:**  
+- **does_each_zone_have_only_one_terminal**: The function returns TRUE if each zone input to this function only has one terminal unit. It returns FALSE if any zone has more than one terminal unit.   
+ 
+**Function Call:**  None    
+
+## Logic: 
+- Set does_each_zone_have_only_one_terminal = TRUE: `does_each_zone_have_only_one_terminal = TRUE`  
+- For each zone id in zone_id_list: `For zone_id in zone_id_list:`   
```

Comment:
Based on a recent exchange (below), exhaust fan may be modeled via a dedicated terminal:

Maria: We do not seem to have a field in the RMD to define zonal exhaust fans. These are important for verifying types of fans in the baseline and that the fan power is allocated to these fans in the same proportion as in the proposed design.
SG: Zone has a ‘exhaust airflow rate’, and ‘terminals’. Would it be correct to specify the exhaust airflow rate to the zone.exhaust_airflow_rate data element and then use terminals to connect it with a terminal that only includes a fan?
The fan would then be connected to a fan system that categorizes it as an exhaust fan